docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -722,10 +642,6 @@ The following table lists all the server state attributes the client can receive
 | `MC_PLAYBACK_STATE_PLAYING`      | Playback state of playing                |
 | `MC_PLAYBACK_STATE_PAUSED`       | Playback state of paused                 |
 | `MC_PLAYBACK_STATE_STOPPED`      | Playback state of stopped                |
-| `MC_PLAYBACK_STATE_NEXT_FILE`    | Playback state of next file              |
-| `MC_PLAYBACK_STATE_PREV_FILE`    | Playback state of previous file          |
-| `MC_PLAYBACK_STATE_FAST_FORWARD` | Playback state of fast forward           |
-| `MC_PLAYBACK_STATE_REWIND`       | Playback state of rewind                 |
```

Comment:
Question. What is the difference between this `MC_PLAYBACK_STATE_REWIND` and `MC_PLAYBACK_STATE_REWIND` on line 732?

Reply:
It is mistyping. So I changed 'MC_PLAYBACK_STATE_REWIND' to 'MC_PLAYBACK_STATE_REWINDING'. Thanks for your reviewing.